docs/reference/ml/anomaly-detection/ml-configuring-categories.asciidoc

```diff
@@ -98,6 +98,28 @@ to disregard multiple sections of the categorization field value. In this
 example, you might create a filter like `[ "\\[statement:.*\\]"]` to remove the
 SQL statement from the categorization algorithm.
 
+[discrete]
+[[ml-per-partition-categorization]]
+== Per-partition categorization
+
+If you enable per-partition categorization, categories are determined
+independently for each partition. For example, if your data includes messages
+from multiple types of logs (such as access logs and error logs), you can use a
+field like the ECS {ecs-ref}/ecs-event.html[`event.dataset` field] as the 
+`partition_field_name` and categorize the messages for each log separately.
+
+If your job has multiple detectors, every detector that uses the `mlcategory`
+keyword must also define a `partition_field_name`. You must use the same
+`partition_field_name` value in all of these detectors. Otherwise, when you
+create or update a job and enable per-partition categorization, it fails.
```

Comment:
Could this be a single sentence?

```
If your job has multiple detectors, every detector that uses the `mlcategory`
keyword must also use the same `partition_field_name`. 
```

Reply:
@sophiec20 I think I'll leave it as two sentences, since IMO the single sentence leaves it unclear whether the partition_field_name is mandatory or whether it's only mandatory that it has the same value.